### ISSUE TITLE
Will this work on an iMac from 2011 that runs Windows 11?

### ISSUE THREAD

**Issue by VirtualMachineMan** [alvr-org/ALVR#1458]
### Please don't delete this.
I know this isn't an issue or a bug report but I've put a lot of time into typing this. It's taken me around 20 minutes just to make this, read it and then edit mistakes and cut boring bits out.

### Don't mention the Oculus app.
I installed it and it says that it couldn't reach the Oculus service or something, and I tried a couple of fixes I found by googling, but they didn't work.

ANYWAY, moving onto the actual question:

I didn't have time this morning (in my timezone) to test SteamVR out, but I got ALVR set up. I am using an iMac 2011, that has Windows 11 on it that uses the integrated AMD 6770M something (can't remember the rest) graphics card. Will the most basic VR game work on it with ALVR?

I know that Minecraft set at a 1920x1080 resolution works super well up to 13 chunks. I've noticed on other games, that don't have code like Minecraft's magic renderer does, it can be super bad or quite good after adjusting settings. The first game I hope to play is Bonelab, which runs OK on the Quest 2 (but not good enough), which is why I want to try it in VR!

So, will the most simple non-resource intensive SteamVR VR game + ALVR work on an iMac from 2011? And will Bonelab run OK at the right settings (probably not)? 

Oh and by the way, at the time of posting this, it is currently 11:33 PM so I'll probably be asleep if someone replies in 1-4 hours.

**Comment by galister** [comment]
I doubt you'd have an enjoyable VR experience with a GPU that has 1GB of video memory (less than the Quest), but you might as well go ahead and try some of the free-to-play VR titles like VRChat or ChilloutVR or the such.

Your GPU also doesn't seem to have encoding support, so you'd be stuck using software encoding, which will likely not give you the quality you're looking for. I think you'll find more enjoyment just using your Quest 2 as standalone.

Regardless, I'm curious to hear how it turns out.

**Comment by StupidRepo** [comment]
Hi! Here is what happened:
* I ran ALVR and everything was epic. SteamVR detected the controllers and headset fine first time. Speaking of the first time, it was very buggy and all I saw was just a light blue colour with some pixels glitched.
* I quit SteamVR which exited ALVR, and I restarted ALVR. SteamVR launched, again as usual. This time, I got to see 'SteamVR - Waiting...' in the headset! (Or it was 'Loading...', can't remember.)
* The tracking was all over the place, it was spazzing out by itself without me moving anything. After a couple seconds of that SteamVR loading screen, the ALVR (client or server, I couldn't tell) gave up and all I saw was just the last frame. SteamVR loading.
* I haven't tried ALVR since this morning but I'll try it again tomorrow morning with, hopefully, more time to fix stuff.

Also, 1GB of video memory? I swear it's more than that. Also, Ubuntu and Windows report the graphics card as a 6770M and not a 6970M.

Wait:
<img width="547" alt="Screenshot 2023-02-15 at 19 58 30" src="https://user-images.githubusercontent.com/69256931/219139212-2e9d2fd0-b989-4fc0-be10-f0e44e6c8968.png">
Damn it. 512MB. The 6970M isn't to be seen anywhere in any settings on Ubuntu or Windows, so that's weird.
<img width="586" alt="Screenshot 2023-02-15 at 19 58 45" src="https://user-images.githubusercontent.com/69256931/219139253-2c22dadd-1189-49a2-bfcb-99d06fd836b6.png">
